webpack/js/app.js

```diff
@@ -5,17 +5,16 @@ import "bootstrap";
 function initDownloadButton() {
   const downloadButton = document.querySelector(".download-btn");
   if (downloadButton) {
-    fetch("https://api.github.com/repos/lektor/lektor/releases", {
-      method: "GET",
-    })
+    // See https://warehouse.pypa.io/api-reference/json.html#project
+    fetch("https://pypi.org/pypi/lektor/json", { method: "GET" })
       .then((res) => res.json())
-      .then((releases) => {
-        const tag = releases[0].tag_name;
+      .then((project) => {
+        const version = project.info.version;
         const link = downloadButton.querySelector("a");
         if (link) {
           const span = document.createElement("span");
           span.className = "version";
-          span.innerText = tag;
+          span.innerText = "v" + version;
```

Comment:
Apparently, in the past, the GitHub releases were named things like ["Lektor 3.1"](https://github.com/lektor/lektor/releases/tag/3.1).

Maybe prefix the version with "Lektor " rather than "v"?  (So that the button text reads "Download Lektor X.YY.Z").

Hmm.  Maybe we should move the "Lektor" into the static HTML, so that the button reads "Download Lektor" before it gets mangled by the javascript (and "Download Lektor X.YY.Z" after mangling).

Reply:
will do